contributors/devel/sig-scheduling/scheduling_code_hierarchy_overview.md

```diff
@@ -66,6 +66,22 @@ Consists of the following four steps ran in the same order:
 In case of processing of any of the extension points fails, `Unreserve` operation
 of all `Reserve` plugins is invoked (e.g. free resources allocated for a gang of pods).
 
+## Scheduler extensibility
```

Comment:
This deserves a standalone document. I can compose one later. For now, just remove this section (as well as "Modifying policies") to keep this document unchanged.